firebase_admin/dynamic_links.py

```diff
@@ -41,40 +38,39 @@
 EVENT_TYPE_APP_FIRST_OPEN = 'app_first_open'
 EVENT_TYPE_APP_RE_OPEN = 'app_re_open'
 
-StatOptions = namedtuple('StatOptions', ['duration_days'])
-
 def get_link_stats(short_link, stat_options, app=None):
-    """ Returns a LinkStats object with the event stats for the given short link
+    """ Returns a `LinkStats` object with the event statistics for the given short link
```

Comment:
I'm pretty sure you need double backticks to make code formatting work in Sphinx. Please verify and fix accordingly.

Reply:
Yes, single backticks are italic